docs/prover/cli/options.md

```diff
@@ -596,45 +632,13 @@ When you suspect you have an old installation. To install the newest version, us
 
 `certoraRun --version`
 
-### `--typecheck_only`
-
-**What does it do?**  
-Stops after running the Solidity compiler and type checking of the spec, before submitting the verification task.
-
-**When to use it?**  
-If you want only to check your spec, or include it in an automated task (e.g., a git `pre-commit` hook).  
-**Example**
 
-`certoraRun Bank.sol --verify Bank:bank.spec --typecheck_only`
 
 Advanced options
 ----------------
 
-(--cloud)=
-### `--cloud`
```

Comment:
You removed this header and its reference, so all usages of that reference ned to be updated.